app/operations/users/destroy_self.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+module Users
+  class DestroySelf < Actor
+    input :user, type: User
+    input :password, type: String
+
+    def call
+      return fail!(error: "Wrong password", status: :unauthorized) unless valid_password?
+
+      ActiveRecord::Base.transaction { user.destroy_fully! }
+
+      true
+    rescue StandardError => e
```

Comment:
Is there a reason for rescuing from `StandardError`?

Is this line being tested?

Usually, it is not good to rescue from a  generic error like this